consensus/reactor.go

```diff
@@ -91,9 +91,9 @@ func (conR *ConsensusReactor) GetChannels() []*p2p.ChannelDescriptor {
 			SendQueueCapacity: 100,
 		},
 		&p2p.ChannelDescriptor{
-			ID:                 DataChannel,
-			Priority:           2,
```

Comment:
I didn't get a good sense actually, so doubtful much. i was thinking of making these configurable to test but didn't get around to it. i can change it back.